### ISSUE TITLE
How can dump all hash160, and address with unspent balances ?

### ISSUE THREAD

**Issue by agoora** [dcousens/fast-dat-parser#9]
Please can you tell me, how command can be used to dump all hash160, address with unspent balances ?
Thanks you.

**Comment by agoora** [comment]
Please, nobody can reply  ?
Thank you.

**Comment by dcousens** [comment]
Hi @agoora,  at this time there isn't a transform that solely deals with a UTXO set.
You can transform the required data using the [`leveldb`](https://github.com/dcousens/fast-dat-parser/blob/master/src/leveldb.hpp),  but,  it has a lot of data you probably aren't interested in.

**Comment by agoora** [comment]
Please can you tell me, how command can be used to dump all hash160 with btc address ? 
Thanks again.

**Comment by dcousens** [comment]
@agoora there isn't a command.

**Comment by agoora** [comment]
Yes, I want to dump all hash160 with btc address, like old parser :
https://github.com/znort987/blockparser
And I do not understand how I can do with your "fast-dat-parser" ...

**Comment by agoora** [comment]
Please let me know how can I use "1 - Outputs every script prefixed with a uint16_t length" to dump all hash160 ?
Thank you .

**Comment by agoora** [comment]
Thanks for the reply.
Unfortunately, I do not know code  , maybe you help me with a tool or a python script, ready to use and that could do this conversion.
I count on your precious help.
Thank you,
Marc.

**Comment by dcousens** [comment]
@agoora,  I do not have the time to do this for you.
There might be an option for such specific functionality at a later date,  but at this time,  you'll need to do some of the hard yards to make it happen using `fast-dat-parser`.

Cheers,

Dan

**Comment by agoora** [comment]
Hi Dan,
"unspents transform" could extract hash160 with unspent balances ?
Thanks again.

**Comment by dcousens** [comment]
@agoora if you are referring to https://github.com/dcousens/fast-dat-parser/commit/600723a4008f58ef2c35649f57a99cdd97c16705,  it isn't ready/tested yet,  but certainly it may give you an idea of how you could do it.

**Comment by agoora** [comment]
Very good functionality, I encourage you to finish it as quickly as possible 👍 
Will it be able to extract  hash160, and btc address with unspent balances?
What command can be executed?
Please keep me informed when it is ready and can be tested.

**Comment by rawmaterials** [comment]
+1

**Comment by agoora** [comment]
Happy New Year !
I thought Santa Claus came with a gift to be able to easily extract the hash160 ....:)
When would this option be available?
Let ma know.
Thanks.

**Comment by agoora** [comment]
Specifically, how could I quickly calculate this hash160?
Please be precise in your explanation.
Thanks.
